modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Hint.js

```diff
@@ -10,57 +10,44 @@
  */
 
 import ClayIcon from '@clayui/icon';
+import ClayPopover from '@clayui/popover';
 import className from 'classnames';
 import PropTypes from 'prop-types';
 import React, {useRef, useState} from 'react';
 
-import Popover from './Popover';
-
-export default function Hint({align, message, position, secondary, title}) {
+export default function Hint({message, position = 'top', secondary, title}) {
 	const iconRef = useRef();
```

Comment:
As it is no longer being used, we can remove it now.

Reply:
Sure! I didn't remove it from the trigger span so SF was not complaning :P